README.md

```diff
@@ -162,6 +167,52 @@ the generic documentation for more information on the process.
    $ kubectl scale --replicas=1 -n kube-system deployment/sealed-secrets-controller
    ```
 
+### Migration from SealedSecret <0.8.0 to 0.8.0+
+
+In versions prior to 0.8.0 the controller was installed into
+`kube-system` namespace by default but to support managed services like
+GKE the default was changed to `sealed-secrets`.
+[Issue ref](https://github.com/bitnami-labs/sealed-secrets/issues/90)
+
+To migrate, follow this procedure:
+
+1. Backup your existing encryption private key:
+   ```sh
+   $ kubectl get secret -n sealed-secrets sealed-secrets-key -o yaml >master.key
+   ```
+
+2. Prepare to load your encryption private key into the new namespace:
+   ```sh
+   $ sed -i 's/kube-system/sealed-secrets/g' master.key
+   ```
+   Or on Mac OS:
+   ```sh
+   $ sed -i '' 's/kube-system/sealed-secrets/g' master.key
```

Comment:
Really, osx doesn't have a "normal" `sed`?  Perhaps we can do `sed <master.key >moved.key` or something that works on both platforms..